.github/dependabot.yml

```diff
@@ -16,6 +16,7 @@ updates:
     - dependency-name: "github.com/aws/*" # updates are too frequent
     - dependency-name: "antrea.io/ofnet"
     - dependency-name: "antrea.io/libOpenflow"
+    - dependency-name: "github.com/ClickHouse/clickhouse-go/v2" # auto-upgrade involves dependency conflicts
```

Comment:
I think `/v2` can be removed? @antoninbas

Reply:
I am not sure actually. Because of how it works in Go, the `v2` is actually part of the module name: https://github.com/ClickHouse/clickhouse-go/blob/f5051e3d27e9f795670d76b203717958f0f1a536/go.mod#L1

There isn't exactly a lot of information in the dependabot documentation...

If we do remove it, we may need a wildcard?

I am happy with keeping the dependency name as it is. I feel like we could add `update-types: ["version-update:semver-major", "version-update:semver-minor"]` so we still potentially get patches. (`"version-update:semver-major"` would not really have an effect here).

Reply:
update-types added